src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -352,6 +322,7 @@ SELECT count(*) FROM pg_dist_partition WHERE colocationid IN (4, 5);
      0
 (1 row)
 
+DROP TABLE table1_group1, table2_group1, table3_group2, table4_group2, table5_groupX, table6_append;
```

Comment:
This file was a bit messy due to the master_create_distributed_table giving invalid colocation ID, but dropping the tables seemed to give a decent balance between preserving coverage and minimizing test changes.